### PR TITLE
f-checkout@3.6.2 - Revert v3.0.0

### PR DESCRIPTION
**Pull request by FayeCarter**
### Fixed
- Spinner and loading state added back in.

### CODE REVIEW

File: packages/components/pages/f-checkout/src/components/Checkout.vue [hunk 1]
```diff
@@ -798,6 +817,28 @@ export default {
...
+
+        /**
+         * Sets the submitting state of the Checkout form. When true a spinner is displayed on the submit button
+         * This is done to allow us to test the setting of this value and ensure it is called with the correct value in the correct order.
+         * @param  {boolean} isFormSubmitting  - whether the form should be in a submitting state or not.
+         */
+        setSubmittingState (isFormSubmitting) {
+            this.isFormSubmitting = isFormSubmitting;
+        },
+
```
Comment (minrwhite) [inline packages/components/pages/f-checkout/src/components/Checkout.vue]: This function doesn't match up with #1404 and doesn't seem to be referenced anywhere else in the PR - I'm assuming this was left in from another changeset?

**Review by minrwhite — COMMENTED** [review]
One comment, but everything else looks good

File: packages/components/pages/f-checkout/src/components/Checkout.vue (hunk 1 above)
Comment (FayeCarter) [reply packages/components/pages/f-checkout/src/components/Checkout.vue]: Ah yes, this should be in a different component now. Thanks! I'll remove it

Comment (FayeCarter) [reply packages/components/pages/f-checkout/src/components/Checkout.vue]: Removed Thanks!

**Review by minrwhite — APPROVED** [review]

**Review by BanksySan — APPROVED** [review]

**Pull request merged**